sample.tf

```diff
@@ -214,12 +214,7 @@ resource "artifactory_keypair" "some-keypairRSA" {
   private_key = file("samples/rsa.priv")
   public_key  = file("samples/rsa.pub")
   alias       = "foo-aliasfoo"
-  lifecycle {
-    ignore_changes = [
-      private_key,
-      passphrase,
-    ]
-  }
+  passphrase  = "some-passphrase"
```

Comment:
Why we shouldn't ignore `private_key`?

Reply:
Both `passphrase` and `private_key` are not updated from Artifactory. So there's no need to ignore them at all.

Reply:
https://github.com/jfrog/terraform-provider-artifactory/blob/master/pkg/artifactory/resource/security/resource_artifactory_keypair.go#L200